atomic_reactor/schemas/container.json

```diff
@@ -254,6 +254,31 @@
           "description": "Relative path to directory containing operator manifests",
           "type": "string",
           "pattern": "^[^/].*$"
+        },
+        "repo_replacements": {
+          "description": "Additional replacements for repos not available in OSBS site config",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "registry": {
+                "description": "Configure replacements for this registry",
+                "type": "string"
+              },
+              "package_mappings": {
+                "description": "Repo replacement mappings",
+                "type": "object",
+                "patternProperties": {
+                  ".*": {"type": "string"}
+                },
+                "examples": [
+                  {"foo-bar": "foo/bar", "spam-eggs": "spam/eggs"}
+                ]
+              }
+            },
+            "required": ["registry", "package_mappings"],
```

Comment:
It shouldn't be required

Reply:
I think it should. The `repo_replacements` key itself is not required, or it can be `repo_replacements: []`, but if you put an item in this list, that item should have both `registry` and `package_mappings`, or it does not make much sense

Reply:
ah right